Anna Gamzinova/HW10a_gamzinova.py

```diff
@@ -0,0 +1,175 @@
+class Company:
+    """
+    Class Company that contain name, industry, employees, headquarter, product and services.
+    """
+
+    def __init__(self, name: str, industry: set, employees: int, headquarter: str, product: set, service: set):
+        self.__name = name
+        self.__industry = industry
+        self.__employees = employees
+        self.__headquarter = headquarter
+        self.__product = product
+        self.__service = service
+
+    @property
+    def name(self):
+        """
+        A method to get the company's name
+        """
+        return self.__name
+
+    @name.setter
+    def name(self, new_name: str):
+        """
+        A method to change the company's name
+        """
+        if len(new_name):
+            self.__name = new_name
+        else:
+            print("Your name is empty, can not update")
+
+    @property
+    def industry(self):
+        """
+        A method to get the company's industry
+        """
+        return self.__industry
+
+    @industry.setter
+    def industry(self, new_industry: str):
+        """
+        A method to update the company's industry
+        """
+        if new_industry in self.__industry:
+            print(f"{new_industry} can not be added")
+        elif not len(new_industry):
+            print("Your new industry is empty, can not update it")
+        else:
+            self.industry.add(new_industry)
+
+    @property
+    def employees(self):
+        """
+        A method to get the company's number of employees
+        """
+        return self.__employees
+
+    @employees.setter
+    def employees(self, new_employees_number: int):
+        """
+        A method to update company's employees number
+        """
+        if new_employees_number > -1:
+            self.__employees = new_employees_number
+        else:
+            print("The number of employees can not be negative")
+
+    def add_employee(self, new_employee_number: int = 1):
+        """
+        A method to add the number of employees to the existing one
+        """
+        self.__employees += new_employee_number
```

Comment:
what happened if I pass -10000 to this method?

Reply:
@BohdanDovbysh1994 
Fixed. 
Thanks a lot for your comment, I knew that I needed a verification there, I probably missed it.